### PR TITLE
#925 AUDIT - start_a_blank_CASE_NOTE()

### PR DESCRIPTION
**Pull request by MDR22**
-Removed unnecessary msgbox commented out
-Changed CASE/NOTE edit mode check to evaluate based on the panel name rather than mode and CASE/NOTE panel 
-Add option to cancel script and submit error report

### CODE REVIEW

**Review by MeganGeissler — APPROVED** [review]
Tested and looks good!

File: MASTER FUNCTIONS LIBRARY.vbs [hunk 1]
```diff
@@ -12221,23 +12221,24 @@ function start_a_blank_CASE_NOTE()
...
-            'msgbox "The script can't open a case note. Reasons may include:" & vbnewline & vbnewline & "* You may be in inquiry" & vbnewline & "* You may not have authorization to case note this case (e.g.: out-of-county case)" & vbnewline & vbnewline & "Check MAXIS and/or navigate to CASE/NOTE, and try again. You can press the STOP SCRIPT button on the power pad to stop the script."
-            BeginDialog Inquiry_Dialog, 0, 0, 241, 115, "CASE NOTE Cannot be Started"
-              ButtonGroup ButtonPressed
-                OkButton 185, 95, 50, 15
-              Text 10, 10, 190, 10, "The script can't open a case note. Reasons may include:"
-              Text 20, 25, 80, 10, "* You may be in inquiry"
-              Text 20, 40, 185, 20, "* You may not have authorization to case note this case (e.g.: out-of-county case)"
-              Text 5, 70, 225, 20, "Check MAXIS and/or navigate to CASE/NOTE, and try again. You can press the STOP SCRIPT button on the power pad to stop the script."
-            EndDialog
+		EMReadScreen case_note_open_check, 8, 1, 72
```
Comment (C-Love) [inline MASTER FUNCTIONS LIBRARY.vbs:12224]: Could we add a read of the mode and the message at the bottom of the screen? Maybe also the PW? Then if it fails and the error report is selected, we can add it to the 'script_run_lowdown' variable so we get additional information in any error report.

**Review by C-Love — CHANGES_REQUESTED** [review]
This all looks great - I only have a suggestion to give us better information if errors are reported in this functionality.

File: MASTER FUNCTIONS LIBRARY.vbs (hunk 1 above)
Comment (MDR22) [reply MASTER FUNCTIONS LIBRARY.vbs:12224]: Good idea. Added script_run_lowdown variable

File: MASTER FUNCTIONS LIBRARY.vbs [hunk 2]
```diff
@@ -12221,23 +12221,34 @@ function start_a_blank_CASE_NOTE()
...
+		If case_note_open_check <> "FMCAMAM2" then
+            EMReadScreen mode_error_check, 1, 20, 9
+			EMReadScreen PW_error_check, 6, 20, 9
```
Comment (C-Love) [inline MASTER FUNCTIONS LIBRARY.vbs:12227]: I think this should be EMReadScreen PW_error_check, 6, 21, 15

**Review by C-Love — CHANGES_REQUESTED** [review]
Looks good except the coordinates for reading the PW.

**Review by C-Love — APPROVED** [review]
Functionality looks good but the coordinates for reading the PW are off. Once that is resolved, you are good to go.

**Pull request merged**